ntoskrnl/mm/balance.c

```diff
@@ -39,6 +39,16 @@ static LONG PageOutThreadActive;
 
 /* FUNCTIONS ****************************************************************/
 
+static inline VOID UpdatePeakCommitment(VOID)
+{
+    // HACK: MmTotalCommittedPages should be adjusted consistently with
+    // other counters at different places.
```

Comment:
Is this comment still relevant? If yes, which other counters/places would you have in mind, for example?

Reply:
Was added by @HBelusca . @HBelusca : What's your view on this comment now ?

Reply:
My view: Since we are not experts in Mm, and this PR isn't an actual serious attempt at getting all the Mm counters consistently implemented, the purpose of this comment is to tell any future Mm-knowledgeable dev to double-check and possibly fix where MmTotalCommittedPages (and the rest) needs to be updated.

Reply:
OK so let's keep this comment anyway